sycl/source/detail/queue_impl.cpp

```diff
@@ -454,6 +509,11 @@ detail::EventImplPtr queue_impl::submit_kernel_direct_impl(
         CodeLoc));
     CommandGroup->MIsTopCodeLoc = IsTopCodeLoc;
 
+    if (auto GraphImpl = getCommandGraph(); GraphImpl) {
```

Comment:
```suggestion
    if (auto GraphImpl = getCommandGraph()) {
```
Just a non-blocking nitpick, might be just my personal preference though.